coalib/coala_main.py

```diff
@@ -69,6 +70,11 @@ def run_coala(log_printer=None,
         dtag = str(sections['default'].get('dtag', None))
         config_file = os.path.abspath(str(sections["default"].get("config")))
 
+        cache = Cache(
+            log_printer,
+            config_file,
+            sections['default'].get('flush_cache', False))
```

Comment:
IMO it's an overkill. Plus performance wise we would be reading and writing as many times.